server/prisma/generator/factories/events.factory.ts

```diff
@@ -83,52 +83,6 @@ const createEvents = async (
         return prisma.event_sponsors.create({ data: eventSponsorData });
       }),
     );
-
-    const tagNames = [
-      'GraphQl',
-      'NodeJs',
-      'JavaScript',
-      'TypeScript',
-      'HTML',
-      'CSS',
-      'Cypress',
-      'Tailwind',
-      'Sass',
-      'BootStrap',
-      'React',
-      'Vue',
-      'NextJs',
-      'NuxtJs',
-      'Angular',
-      'Svelete',
-      'SveleteKit',
-      'Vite',
-      'Prisma',
-      'Ruby',
-      'Rust',
-    ];
-    const tagsCount = Math.round((Math.random() * tagNames.length) / 5);
-
-    const selectedTags = randomItems(tagNames, tagsCount, true);
-    const connectOrCreateTags = selectedTags.map((name) => ({
-      tag: {
-        connectOrCreate: {
-          where: { name },
-          create: { name },
-        },
-      },
-    }));
-
-    await prisma.events.update({
-      where: { id: event.id },
-      data: {
-        tags: {
-          create: connectOrCreateTags,
-        },
-      },
-    });
-
-    events.push(event.id);
```

Comment:
I don't know why I have deleted factory, I guess I was on frenzy. anyway should they be brought back or when we try to use EventTags in the future we will use them?